### PR TITLE
Fix lightFX being off center in some rotations

### PR DESCRIPTION
**Pull request by WantDiscussion**
Fix lightFX being off center in some rotations

### CODE REVIEW

File: src/openrct2/drawing/LightFX.cpp [hunk 1]
```diff
@@ -674,30 +674,30 @@ void lightfx_add_3d_light(uint32_t lightID, uint16_t lightIDqualifier, int16_t x
...
-            x += 16;
-            y -= 16;
-            break;
-        default:
-            return;
-    }
+    int16_t x = mapPosition.x + offsetX + 16;
+    int16_t y = mapPosition.y + offsetY + 16;
+
+    // switch (get_current_rotation())
```
Comment (Gymnasiast) [inline src/openrct2/drawing/LightFX.cpp]: Never comment out code. Remove it entirely. We have source control if we ever want it back.

**Comment by Gymnasiast** [comment]
Could you post a before and after screenshot?

**Comment by WantDiscussion** [comment]
> 
> 
> Could you post a before and after screenshot?

Before:
Annotation 2020-03-19 001035

After:
Annotation 2020-03-19 001125

**Comment by Gymnasiast** [comment]
Very good!

Could you also add an entry to the changelog?
`- Fix: [#10963] Light effects are drawn off-centre in some rotations.`

**Comment by duncanspumpkin** [comment]
Yeah I noticed this before but guessed it was there for a reason.

**Pull request merged**